apps/dashboard/app/models/user_configuration.rb

```diff
@@ -54,6 +57,11 @@ class UserConfiguration
     ConfigurationProperty.property(name: :custom_css_files, default_value: []),
 
     ConfigurationProperty.property(name: :dashboard_title, default_value: 'Open OnDemand', read_from_env: true),
+
+    # Navigation properties
+    ConfigurationProperty.with_boolean_mapper(name: :show_all_apps_link, default_value: false, read_from_env: true, env_names: ['SHOW_ALL_APPS_LINK']),
+    ConfigurationProperty.with_boolean_mapper(name: :navconfig_categories_filter, default_value: NavConfig.categories_whitelist?),
```

Comment:
While we're renaming this, I wonder if `navconfig_categories_filter` is the best word.  I don't have a better one at this time, but `filter` does seem close.